src/coverlet.core/Extensions/DependencyInjectionExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Coverlet.Core.Attributes;
+
+namespace coverlet.core.Extensions
+{
+    public static class DependencyInjectionExtensions
+    {
+        [ExcludeFromCoverage]
+        public static T GetService<T>(this IServiceProvider serviceProvider)
+        {
+            return (T)serviceProvider.GetService(typeof(T));
```

Comment:
Why `[ExcludeFromCoverage]`? core lib won't be instrumented so we don't need to exclude, you can remove it.

Reply:
Fixed, I added it as it was on `HelperExtensions`. Maybe if core lib won't be instrumented I should remove `[ExcludeFromCoverage]` from ``HelperExtensions` too?

Reply:
Ah ok...understood...I think it was placed to remove from "autocoverage" that we do on build.